### DISCUSSION TITLE
Tag version different from package.json version?

### DISCUSSION THREAD

**Discussion by mjoffily** [semantic-release/semantic-release#2184] [Q&A] [answered]
Just trying to clarify and confirm if this is expected behaviour...
Once a release is created, a TAG is created and pushed, but the version in the package.json is still the old one...
Shouldn't they match? 

![image](https://user-images.githubusercontent.com/10581006/136898853-e8285052-a58b-4002-b416-829e8735e8b5.png)

**Comment by JoA-MoS** [comment, marked as answer]
This is by design - https://semantic-release.gitbook.io/semantic-release/support/faq#why-is-the-package.jsons-version-not-updated-in-my-repository

**Reply by mjoffily** [reply]
Got it, thanks for clarifying.
